src/stirling/source_connectors/socket_tracer/testing/containers/thriftmux/Dockerfile

```diff
@@ -0,0 +1,37 @@
+# Copyright 2018- The Pixie Authors.
```

Comment:
We'd preferably not have a Dockerfile to build this image. Looks like you're trying to build a server image with the scala client and server.

Have you looked into using bazel itself to create the images. A useful reference is here: https://github.com/bazelbuild/rules_docker#scala_image

If we do that, then we won't need to host an image on docker or GCR, because bazel will build it natively and we can load the tarball for the image natively in the test. 

Another useful resource is in `src/stirling/source_connectors/socket_tracer/protocols/http2/testing/go_grpc_server/BUILD.bazel` where you can see we do this sort of thing. In that case, we're building a go server that we use in tests, but it's essentially the same thing but with Scala here.

Reply:
I actually tried to use bazel initially. I think I ran into some trouble installing maven dependencies with the scala binary so I opted for something that I was more familiar first.

I'll give the native bazel build and a scala image another go.